docs/gpa.md

```diff
@@ -233,78 +248,100 @@ The information included in the notice of intended procurement covers all the in
     <tbody>
       <tr class="section">
         <td>X:7</td>
-        <td colspan="2">A procuring entity shall make available to suppliers tender documentation that includes all information necessary to permit suppliers to prepare and submit responsive tenders. Unless already provided in the notice of intended procurement, such documentation shall include a complete description of:</td>
-      </tr>
+        <td colspan=2>A procuring entity shall make available to suppliers tender documentation that includes all information necessary to permit suppliers to prepare and submit responsive tenders. Unless already provided in the notice of intended procurement, such documentation shall include a complete description of:</td>
+     </tr>
       <tr>
         <td>X:7(a)</td>
         <td>the procurement, including the nature and the quantity of the goods or services to be procured or, where the quantity is not known, the estimated quantity and any requirements to be fulfilled, including any technical specifications, conformity assessment certification, plans, drawings or instructional materials;</td>
         <td markdown=1>
 
-* 
+* For *the procurement, including the nature and the quantity of the goods or services to be procured or, where the quantity is not known, the estimated quantity*, follow the guidance for article VII:2(b)
+* For *any requirements to be fulfilled, including any technical specifications, conformity assessment certification, plans, drawings or instructional materials*, follow the guidance for article VII:2(j)
 </td>
       </tr>
       <tr>
         <td>X:7(b)</td>
         <td>any conditions for participation of suppliers, including a list of information and documents that suppliers are required to submit in connection with the conditions for participation;</td>
         <td markdown=1>
 
-* 
+* Please follow the guidance for VII:2(j)
 </td>
       </tr>
       <tr>
         <td>X:7(c)</td>
         <td>all evaluation criteria the entity will apply in the awarding of the contract, and, except where price is the sole criterion, the relative importance of such criteria;</td>
         <td markdown=1>
 
-* 
+* Please follow the guidance for VII:(k)
```

Comment:
I assume you mean VII:2(k)? But (k) is about selection criteria. This is about award criteria.

Reply:
Oops, nevermind – the GitHub preview showed a different context on the other screen. Just need to correct the typo:

```suggestion
* Follow the guidance for VII:2(k)
```

Reply:
Done